requirements.txt

```diff
@@ -19,8 +19,10 @@ charset-normalizer==2.0.12
 cryptography==36.0.1
     # via
     #   django-rest-knox
+    #   duo-universal
     #   jwcrypto
     #   pyjwt
+    #   pyopenssl
```

Comment:
⚠️ From  https://pypi.org/project/pyOpenSSL/ ⚠️ 

"Note: The Python Cryptographic Authority strongly suggests the use of [pyca/cryptography](https://github.com/pyca/cryptography) where possible. If you are using pyOpenSSL for anything other than making a TLS connection you should move to cryptography and drop your pyOpenSSL dependency."

=> Open an issue on python duo security module ?